web/survey/views.py

```diff
@@ -546,22 +546,15 @@ def external_conflict_view(request):
         run = Run.objects.get(id=run_id)
         conflicts = ExternalConflict.objects.filter(
             detection_id__in=[d.id for d in Detection.objects.filter(run=run)])
-
         if len(conflicts) == 0:
             messages.info(request, "All external conflicts for this run have been resolved")
             return HttpResponseRedirect('/admin/survey/run')
 
-        external_conflict_id = request.GET.get('external_conflict_id', None)
-        if external_conflict_id is None:
-            # Should only be the case when entering the manual inspection view
-            conflict = conflicts[0]
-        else:
-            conflict = ExternalConflict.objects.get(id=external_conflict_id)
-
+        external_conflict_id = int(request.GET.get('external_conflict_id', conflicts[0].id))
```

Comment:
Will there always be a `external_conflict_id`?

Reply:
Yes, it is either in the URL as `external_conflict_id` or it will fetch the id of the first item in the `conflicts` list. If there is nothing in the `conflicts` list then it redirects to the survey page

```
if len(conflicts) == 0:
    messages.info(request, "All external conflicts for this run have been resolved")
    return HttpResponseRedirect('/admin/survey/run')
```